src/components/Toggle.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <fieldset class="f-toggle">
+    <input
+      :id="id"
+      type="checkbox"
+      :checked="value"
+      @change="$emit('update:value', $event.target.checked)" />
+    <label :for="id">
+      <div class="switch" />
+      <span>{{name}}</span>
```

Comment:
please make the switch respond only if you really click on the switch itself and not anywhere on the toggle class

Reply:
Done